examples/example_helper.rb

```diff
@@ -27,6 +28,14 @@ def persistence_conn_string
       ENV['DB_CONN_STRING'] || 'sqlite:/'
     end
 
+    def telemetry_adapter
+      if (host = ENV['TELEMETRY_STATSD_HOST'])
+        Dynflow::TelemetryAdapters::StatsD.new host
+      else
+        Dynflow::TelemetryAdapters::Dummy.new
+      end
+    end
+
     def persistence_adapter
       Dynflow::PersistenceAdapters::Sequel.new persistence_conn_string
```

Comment:
Persistence layer can be easily a bottleneck too, if you can try to inject some measurements into sequel (SQL query duration for example).

Reply:
Good point, I'll try looking into it

Reply:
Which level of detail would make sense? Several options come to mind

- Query duration with details - "There was a SELECT/INSERT/DELETE which took XYms"
- Query duration - "There was an unspecified SQL query which took XYms"
- Duration per execution plan - "When executing an execution plan, Dynflow spent XYms in the database", closest to `:http_request_db_duration` in Foreman

We could probably get even more details but I'm not sure if it would be useful in any way.